hash/sega_beena_cart.xml

```diff
@@ -181,23 +430,24 @@ license:CC0
 		<description>Pocket Monsters Diamond &amp; Pearl Pokemon o Sagase! Meiro de Daibouken!</description>
 		<year>2009</year>
 		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ポケットモンスター ダイヤモンド・パール ポケモンをさがせ!めいろでだいぼうけん!"/>
```

Comment:
This also has a half-width exclamation mark.

Reply:
Replaced all halfwidth with regular chars and removed spaces!